modules/geth/charts.go

```diff
@@ -0,0 +1,239 @@
+			{ID: txPoolQueuedDiscard, Name: "discard"},
+			{ID: txPoolQueuedEviction, Name: "eviction"},
+			{ID: txPoolQueuedNofunds, Name: "no_funds"},
+			{ID: txPoolQueuedRatelimit, Name: "ratelimit"},
+		},
+	}
+	chartP2PNetwork = Chart{
+		ID:    "p2p_network",
+		Title: "P2P bandwidth",
+		Units: "bytes/s",
+		Fam:   "p2p_bandwidth",
+		Ctx:   "geth.p2p_bandwidth",
+		Dims: Dims{
+			{ID: p2pIngress, Name: "ingress", Algo: "incremental"},
+			{ID: p2pEgress, Name: "egress", Mul: -1, Algo: "incremental"},
+		},
+	}
+	chartReorgs = Chart{
+		ID:    "reorgs_executed",
+		Title: "Executed Reorgs",
+		Units: "reorgs",
+		Fam:   "reorgs",
+		Ctx:   "geth.reorgs",
+		Dims: Dims{
+			{ID: reorgsExecuted, Name: "executed"},
+		},
+	}
+	chartReorgsBlocks = Chart{
+		ID:    "reorgs_blocks",
+		Title: "Blocks Added/Removed from Reorg",
+		Units: "blocks",
+		Fam:   "reorgs",
+		Ctx:   "geth.reorgs_blocks",
+		Dims: Dims{
+			{ID: reorgsAdd, Name: "added"},
+			{ID: reorgsDropped, Name: "dropped"},
+		},
+	}
+	// chartP2PNetworkDetails = Chart{
+	// 	ID:    "p2p_eth_65",
+	// 	Title: "Eth/65 Network utilization",
+	// 	Units: "bytes",
+	// 	Fam:   "p2p_eth_65",
+	// 	Ctx:   "geth.p2p_eth_65",
+	// 	Dims: Dims{
+	// 		{ID: p2pIngressEth650x00, Name: "Eth/65 handshake ingress"},
+	// 		{ID: p2pIngressEth650x01, Name: "Eth/65 new block hash ingress"},
+	// 		{ID: p2pIngressEth650x03, Name: "Eth/65 block header request ingress"},
+	// 		{ID: p2pIngressEth650x04, Name: "Eth/65 block header response ingress"},
+	// 		{ID: p2pIngressEth650x05, Name: "Eth/65 block body request ingress"},
+	// 		{ID: p2pIngressEth650x06, Name: "Eth/65 block body response ingress"},
+	// 		{ID: p2pIngressEth650x08, Name: "Eth/65 transactions announcement ingress"},
+	// 		{ID: p2pEgressEth650x00, Name: "Eth/65 handshake egress", Mul: -1},
+	// 		{ID: p2pEgressEth650x01, Name: "Eth/65 new block hash egress", Mul: -1},
+	// 		{ID: p2pEgressEth650x03, Name: "Eth/65 block header request egress", Mul: -1},
+	// 		{ID: p2pEgressEth650x04, Name: "Eth/65 block header response egress", Mul: -1},
+	// 		{ID: p2pEgressEth650x05, Name: "Eth/65 block body request egress", Mul: -1},
+	// 		{ID: p2pEgressEth650x06, Name: "Eth/65 block body response egress", Mul: -1},
+	// 		{ID: p2pEgressEth650x08, Name: "Eth/65 transactions announcement egress", Mul: -1},
+	// 	},
+	// }
+	chartNumberOfPeers = Chart{
+		ID:    "p2p_peers_number",
+		Title: "Number of Peers",
+		Units: "peers",
+		Fam:   "p2p_peers",
+		Ctx:   "geth.p2p_peers",
+		Dims: Dims{
+			{ID: p2pPeers, Name: "Peers"},
+		},
+	}
+
+	chartp2pDialsServes = Chart{
+		ID:    "p2p_dials_serves",
+		Title: "P2P Serves and Dials",
+		Units: "calls/s",
+		Fam:   "p2p_peers",
+		Ctx:   "geth.p2p_peers",
+		Dims: Dims{
+			{ID: p2pDials, Name: "dials", Algo: "incremental"},
+			{ID: p2pServes, Name: "serves", Algo: "incremental"},
+		},
+	}
+	chartRpcInformation = Chart{
+		ID:    "rpc_metrics",
+		Title: "RPC information",
+		Units: "rpc calls",
```

Comment:
```suggestion
		Title: "RPC calls",
		Units: "calls/s",
```

Information is more general, I see this chart is about RPC calls.